src/Controls/samples/Controls.Sample.UITests/Issues/Issue18242.xaml

```diff
@@ -3,36 +3,82 @@
 			 xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
 			 x:Class="Maui.Controls.Sample.Issues.Issue18242"
 			 Title="Issue 18242">
```

Comment:
Maybe this should be a new page? I just finished up the Android fixes and if there are 2 columns this page will be too narrow and cramped for a good test on android/ios

Reply:
Yeah, that's probably a good idea.